src/renderer/reducers/settings.js

```diff
@@ -127,10 +127,17 @@ const defaultsForCurrency: Currency => CurrencySettings = crypto => {
   };
 };
 
+const defaultLanguage: () => string = () => {
+  // Nb If the os language is in the list [fr, ru] (LL-7027) default to it
+  const pushedLanguages = ["fr", "ru"];
```

Comment:
i think we want to use the list of languages supported by the live here so en,fr,es,ru,zh :+1: 
you'll find the exported dataset somewhere in the locales files

Reply:
That's why I referred to the task, it says french and russian for now